### ISSUE TITLE
Mask R-CNN baseline version

### ISSUE THREAD

**Issue by pxliang** [MouseLand/cellpose#568]
I am wondering if you can release the version of Mask R-CNN you used in the paper?

**Comment by carsen-stringer** [comment]
We used https://github.com/matterport/Mask_RCNN version 2.1 and this was the training script: https://github.com/MouseLand/cellpose/blob/main/paper/1.0/train_maskrcnn.py
